api/src/main/kotlin/edu/wgu/osmt/io/xlsx/RichSkillXlsxExport.kt

```diff
@@ -0,0 +1,52 @@
+package edu.wgu.osmt.io.xlsx
+
+import edu.wgu.osmt.config.AppConfig
+import edu.wgu.osmt.jobcode.JobCode
+import edu.wgu.osmt.jobcode.JobCodeBreakout
+import edu.wgu.osmt.richskill.RichSkillAndCollections
+
+class RichSkillXlsxExport(
+    private val appConfig: AppConfig
+) : XlsxResource<RichSkillAndCollections>("RichSkillXlsxExport") {
+    private val listDelimiter = "; "
+
+    override fun columnTranslations(data: List<RichSkillAndCollections>): Array<XlsxColumn<RichSkillAndCollections>> {
+        val columns = arrayOf(
+            XlsxColumn<RichSkillAndCollections>("Canonical URL") { it.rs.canonicalUrl(appConfig.baseUrl) },
+            XlsxColumn("RSD Name") { it.rs.name },
+            XlsxColumn("Authors") { it.rs.authors.map { author -> author.value ?: "" }.joinToString(listDelimiter) },
```

Comment:
What do you think of merging the map call to be used in the joinToString method? something like: 

`XlsxColumn("Authors") { it.rs.authors.joinToString(listDelimiter) { author -> author.value ?: "" } },`

this can be applied to most of the map{} calls in this method.